ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -234,6 +238,7 @@ protected ObjectMapper getJsonMapper() {
     // superclass creates a new json mapper on every call
     // we should probably ony create one per application
     ObjectMapper jsonMapper = super.getJsonMapper();
+    jsonMapper.registerModule(new StructSerializationModule(jsonMapper));
```

Comment:
`StreamedQueryResource` has to do these exact same steps. So does `QueryStreamWriterTest` and `StructSerializationModuleTest`.  Move this _common code_ into a common place.  This common place only needs one copy of the `ObjectMapper`. i.e. create one instance, configure it and then use it in all these places. (e.g. using the code I supplied before).